addons/mail/static/src/main.js

```diff
@@ -25,3 +26,4 @@ registry.category('actions').add('mail.action_discuss', DiscussContainer);
 
 registry.category('main_components').add('DialogManagerContainer', { Component: DialogManagerContainer });
 registry.category('main_components').add('ChatWindowManagerContainer', { Component: ChatWindowManagerContainer });
+registry.category('main_components').add('PopoverManagerContainer', { Component: PopoverManagerContainer });
```

Comment:
Isn't it currently missing from the public page boot?

Reply:
Indeed... I was clearly wondering that on Friday, and forgot to address that. Thanks for recalling it to me, will do a PR right now to fix this.